server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -140,9 +140,9 @@
      * Message for when a pre-computed annotation index cannot be loaded
      * @param index name of the annotation index
      */
-    @LogMessage(level = ERROR)
-    @Message(id = 2, value = "Could not read provided index: %s")
-    void cannotLoadAnnotationIndex(String index);
+    @LogMessage(level = WARN)
+    @Message(id = 2, value = "Loading failed for the annotation index \"%s\" with the following failure message: %s")
```

Comment:
thanks for the explanation. In that case, this string should be more generic like `Loading failed for the annotation index \"%s\" with the following exception: %s`